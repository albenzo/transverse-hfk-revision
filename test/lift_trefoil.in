-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 2
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 2
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 3
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 3
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 4
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 4
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 5
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 5
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 6
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 6
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 7
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 7
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 8
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 8
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 9
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 9
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 10
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 10
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 11
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 11
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 12
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 12
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 13
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 13
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 14 
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 14
-i 5 -X [5,4,3,2,1] -O [3,2,1,5,4] -n 15
-i 5 -X [1,2,3,4,5] -O [4,5,1,2,3] -n 15