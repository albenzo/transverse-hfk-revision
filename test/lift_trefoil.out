theta_2 is null-homologous
theta_2 is NOT null-homologous
theta_3 is null-homologous
theta_3 is NOT null-homologous
theta_4 is null-homologous
theta_4 is NOT null-homologous
theta_5 is null-homologous
theta_5 is NOT null-homologous
theta_6 is null-homologous
theta_6 is NOT null-homologous
theta_7 is null-homologous
theta_7 is NOT null-homologous
theta_8 is null-homologous
theta_8 is NOT null-homologous
theta_9 is null-homologous
theta_9 is NOT null-homologous
theta_10 is null-homologous
theta_10 is NOT null-homologous
theta_11 is null-homologous
theta_11 is NOT null-homologous
theta_12 is null-homologous
theta_12 is NOT null-homologous
theta_13 is null-homologous
theta_13 is NOT null-homologous
theta_14 is NOT null-homologous
theta_15 is null-homologous
theta_15 is NOT null-homologous
